metamorph/plugins/morph_resultsdb.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/python
+import argparse
+import logging
+import logging.config
+import json
+import time
+import os
+import requests
+
+from metamorph.lib.logging_conf import setup_logging
+
+
+class ResultsDBApiException(Exception):
+    pass
+
+
+class ResultsDBApi(object):
+    """
+    Class to communicate and process data with resultsDB
+    """
+    TIMEOUT_LIMIT = 7200  # Wait 2 hours maximally
+    RESULTSDB_LIMITER = 10
+    MINUTE = 60  # 60 seconds
```

Comment:
I don't really think it's necessary to define MINUTE as a class constant. This seems to lead to having to do math on its value instead of just passing in given integers. Is it really any more clear to say sleep(self.MINUTE + self.MINUTE) instead of saying sleep(120) ?